client/src/components/QA/Questions/AnswerModal.jsx

```diff
@@ -91,16 +91,17 @@ class AnswerModal extends React.Component {
       body, name, email, imgFileURLs,
     } = this.state;
 
-    axiosConfig.post(`/qa/questions/${question_id}/answers`, {
-      body, name, email, photos: imgFileURLs,
-    })
-      .then(() => {
-        hideModal();
-        getAnswers();
-      })
-      .catch((err) => {
-        console.log(err);
-      });
```

Comment:
Would you undo this stuff? Thanks for your help earlier

Reply:
think you commented on wrong lines of code. Here's what it looks like now: 
![image](https://user-images.githubusercontent.com/11972548/189235867-fc72a724-31d3-487b-8c1f-6e0d5fed25cd.png)